scipy/optimize/__init__.py

```diff
@@ -110,7 +110,8 @@
 .. autosummary::
    :toctree: generated/
 
-   fixed_point - Single-variable fixed-point solver
+   fixpoint - Single-variable fixed-point solver
+   fixed_point - Steffensen's method for fixed points
```

Comment:
fine, if fixed_point will eventually go away, then there won't be any confusion anymore.